docs/release_notes/ibexa_dxp_v4.6.md

```diff
@@ -0,0 +1,191 @@
+
+For more information, see [Orders block documentation](https://doc.ibexa.co/projects/userguide/en/master/content_management/block_reference/#orders-block).
+
+#### Quick order
+
+The quick order form allows users to streamlines the process of placing orders 
+with multiple items in bulk directly from the storefront.
+Customers don't need to look for products, 
+they can fill in a provided form with products' SKU number and quantity, 
+or upload their own list directly into the system. 
+Quick order form is available to both registered and guest users.
+
+For more information, see [Quick order documentation](https://doc.ibexa.co/en/master/commerce/cart/quick_order/).
+
+#### Shipping management
+
+Shipping management allows you to work with the shipments: view and modify their status, filter shipments in shipment lists and check all the details.
+You can access shipments for your own orders or all the shipments that exist in the system, depending on your permissions.
+
+For more information, see [Work with shipments](https://doc.ibexa.co/projects/userguide/en/master/commerce/shipping_management/work_with_shipments/).
+
+#### Owner criterion
+
+Orders and shipments search now supports user reference:
+
+- `OwnerCriterion` Criterion searches for orders based on the user reference.
+- `Owner` Criterion searches for shipments based on the user reference.
+
+#### Customize checkout workflow
+
+You can create a PHP definition of the new strategy that allows for workflow manipulation. 
+Defining strategy allows to add conditional steps for workflow if needed.
+When a conditional step is added, the checkout process uses the specified workflow and proceeds to the defined step.
+
+For more information, see (https://doc.ibexa.co/en/master/commerce/checkout/customize_checkout/#create-custom-strategy)
+
+### New features and improvements in PIM
+
+#### Virtual products
+
+With this feature, you can create virtual products - non-tangible items such as memberships, services, warranties.
+To create a virtual product, first, you have to create a virtual Product Type.
+Virtual products don’t require shipment when they are purchased without other physical products.
+
+For more information, see [Create virtual products](https://doc.ibexa.co/projects/userguide/en/master/pim/create_virtual_product/.)
+
+#### IsVirtual Criterion
+
+Product search now supports product virutal and physical product type:
+
+- `IsVirtual` - searches for virtual or physical products. 
+
+#### Product page URLs
+
+When you are creating new product type you can set up product URL alias name pattern.
+With this feature, you can also create custom URL and URL alias name pattern field based on product attributes.
+Customized URLs are easy to remember, help with SEO optimization and reduce bounce rates on the website. 
+
+For more information, see [Product page URLs](https://doc.ibexa.co/projects/userguide/en/master/pim/work_with_product_page_urls/).
+
```

Comment:
[WIP]
```suggestion

#### Product migration

[Product variant](importing_data.md#product-variant) and [product asset](importing_data.md#product-asset) can be created through [data migration](data_migration.md).

```

Reply:
Nothing to do w/ perso, misplaced since reordering.

@dabrt Where could I move this entry?

Reply:
Done in 992a8d6095dd1e9982a202c395cc8674d7999fef